pandas/core/resample.py

```diff
@@ -624,18 +624,134 @@ def backfill(self, limit=None):
 
     def fillna(self, method, limit=None):
         """
-        Fill missing values
+        Fill the new missing values in the resampled data using different
```

Comment:
This should be a single line. Does

    Fill missing values introduced by upsampling.

sound good?